client/elements/menus/more-par-menu.html

```diff
@@ -76,20 +86,43 @@
                   },
                   loadingParallels: {
                       type: Boolean,
-                      value: true
+                  },
+                  gotParallels: {
+                      type: Boolean,
+                      value: false,
+                      computed: "_gotParallels(loadingParallels)"
                   }
               }
           }
 
+          _notifyCopy(message, success) {
+              this.dispatchEvent(new CustomEvent('par-menu-copied', {
+                  detail: {message: message, success: success},
+                  bubbles: true,
+                  composed: true
+              }))
+          }
+
           // copy the parallels-table in html-string
           _copyContent() {
-              const copyTextarea = this._computeCopyTable();
-              this._copyToClipboard(copyTextarea);
+              try {
+                  const copyTextarea = this._computeCopyTable();
+                  this._copyToClipboard(copyTextarea);
+                  this._notifyCopy('Parallels table copied!', true);
+              } catch (err) {
+                  this._notifyCopy('Error!', false);
+              }
           }
 
-          _gotParallels() {
-              const copyTextarea = this._computeCopyTable();
-              this._copyToClipboard(copyTextarea);
+          _gotParallels(loadingParallels) {
+              if (loadingParallels) {
+                  return true;
+              }
+              if (Object.keys(this.parallels).length === 0 && this.parallels.constructor == Object) {
+                  return true;
+              } else {
+                  return false;
+              }
```

Comment:
`return loadingParallels || (Object.keys(this.parallels).length === 0 && typeof this.parallels === 'object')`